### ISSUE TITLE
Automated DNS lookups on Firefox startup with HTTPS Everywhere enabled

### ISSUE THREAD

**Issue by username4650** [EFForg/https-everywhere#3354]
HTTPS Everywhere is causing two DNS lookups whenever I start Firefox with the extension enabled. It only occurs when HTTPS Everywhere is enabled. The Observatory feature is turned off. I tried toggling various about:config observatory entries but could not prevent the browser from doing the DNS lookups. I know this possibly has something to do with checking to see if Tor is being used, so that the SSL certificate checks can be sent through Tor, but if the observatory is disabled, it should not be happening. Only the two DNS looksups (on port 53) were noticed. No other connections (e.g. to check.torproject.org) occur, although it's possible others were attempted and blocked by my firewall settings. This seems like it's bad for privacy/tracking/fingerprinting if the same DNS queries are being sent out every time Firefox is started. Is there an about:config setting that can be changed to prevent this, or is this a bug?

**Comment by fuglede** [comment]
Duplicate of #2719.

**Comment by rugk** [comment]
Why not close it if this is a duplicate?
